pyproject.toml

```
@@ -44,13 +44,40 @@ dev = [
     "tox",
 ]
 tests = [
+    "pytest>=6.0",
+    "pytest-split",
+    "pytest-dotenv",
+    "requests-mock",
+    "pytest-cov",
+    "pytest-describe",
```

```suggestion
    "pytest-cov",
```
same here. I am not sure if we need

---

removed. thanks.